de.peeeq.wurstscript/src/main/java/de/peeeq/wurstscript/validation/TRVEHelper.java

```diff
@@ -0,0 +1,7 @@
+package de.peeeq.wurstscript.validation;
+
+import java.util.HashSet;
+
+public class TRVEHelper {
+    public static final HashSet<String> TO_KEEP = new HashSet<>();
```

Comment:
Please don't use mutable static variables. I would prefer to have this as a variable in `ImTranslator`. 

Also the all caps style is usually only used for immutable constants.

Reply:
it doesn't work as variable in ImTranslator because I need to add names from the validator.